Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp

```diff
@@ -95,6 +95,7 @@
 			</div>
 		</c:forEach>
 
+		<c:if test="${editoraccess != '1' || owner == '1' || isProjectAdmin || isProjectContributor}">
```

Comment:
why is the editoraccess checked here? This role has nothing to do with seeing workspaces.